### ISSUE TITLE
php-fcgi no working!

### ISSUE THREAD

**Issue by me11ved** [Optware/Optware-ng#211]
Hi!
I did everything on this article but the application does not work :(
https://www.hqt.ro/nginx-web-server-with-php-support-through-optware-ng/
Go to http://192.168.59.1/info.php
Error:
An error occurred.

Model: RT-AC88U
Firmware: 380.68_2

**Comment by alllexx88** [comment]
Hi, it's working again now: `ipkg update; ipkg upgrade php`. The recent upgrade just disables zen `opcache` module in the default `php.ini`: it clearly doesn't (yet) work with `php-fcgi`

**Comment by alllexx88** [comment]
It's working now, with `opcache` too (via installing `php-opcache` module), so I'm closing the issue

**Comment by me11ved** [comment]
Thanks for the prompt decision of a problem. But I work in conjunction with Apache. This problem is also solved there?

**Comment by alllexx88** [comment]
Yes, it should be fine. The problem was with `opcache` zend extension, not with `php-fcgi` directly, so `php-apache` should be fine too.

**Comment by me11ved** [comment]
I go Apache start and in the errors log:
Failed loading /opt/lib/php/extensions/opcache.so:  (null)

phpMyAdmin - Error:
The mbstring extension is missing. Please check your PHP configuration.
P.S.  module php-mbstring installed

**Comment by alllexx88** [comment]
Please upgrade packages to the latest with `ipkg update; ipkg upgrade`. It works for me now (armv7 router):

```
[root@unknown root]$ ipkg update
Downloading http://ipkg.nslu2-linux.org/optware-ng/buildroot-armeabi-ng/Packages.gz.
Inflating http://ipkg.nslu2-linux.org/optware-ng/buildroot-armeabi-ng/Packages.gz.
Updated list of available packages in /opt/lib/ipkg/lists/optware-ng.
[root@unknown root]$ ipkg install php-apache
Installing php-apache (5.6.31-5) on /opt/.
Downloading http://ipkg.nslu2-linux.org/optware-ng/buildroot-armeabi-ng/php-apache_5.6.31-5_arm.ipk.
Installing apache (2.4.27-1) on /opt/.
Downloading http://ipkg.nslu2-linux.org/optware-ng/buildroot-armeabi-ng/apache_2.4.27-1_arm.ipk.
Installing apr (1.6.2-1) on /opt/.
Downloading http://ipkg.nslu2-linux.org/optware-ng/buildroot-armeabi-ng/apr_1.6.2-1_arm.ipk.
Installing apr-util (1.6.0-1) on /opt/.
Downloading http://ipkg.nslu2-linux.org/optware-ng/buildroot-armeabi-ng/apr-util_1.6.0-1_arm.ipk.
Installing e2fsprogs (1.42.12-1) on /opt/.
Downloading http://ipkg.nslu2-linux.org/optware-ng/buildroot-armeabi-ng/e2fsprogs_1.42.12-1_arm.ipk.
Configuring apr.
Configuring apr-util.
Configuring e2fsprogs.
update-alternatives: Linking //opt/bin/chattr to /opt/bin/e2fsprogs-chattr
update-alternatives: Linking //opt/bin/lsattr to /opt/bin/e2fsprogs-lsattr
update-alternatives: Linking //opt/sbin/fsck to /opt/sbin/e2fsprogs-fsck
Configuring apache.
update-alternatives: Linking //opt/sbin/htpasswd to /opt/sbin/apache-htpasswd
update-alternatives: Linking //opt/sbin/httpd to /opt/sbin/apache-httpd
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 192.168.133.1. Set the 'ServerName' directive globally to suppress this message
httpd (no pid file) not running
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 192.168.133.1. Set the 'ServerName' directive globally to suppress this message
Configuring php-apache.
[root@unknown root]$ ipkg install php-mbstring
Installing php-mbstring (5.6.31-8) on /opt/.
Downloading http://ipkg.nslu2-linux.org/optware-ng/buildroot-armeabi-ng/php-mbstring_5.6.31-8_arm.ipk.
Configuring php-mbstring.
[root@unknown root]$ /opt/sbin/httpd -k restart
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 192.168.133.1. Set the 'ServerName' directive globally to suppress this message
```

Just complains about unset `ServerName`

**Comment by alllexx88** [comment]
OK, `php-apache` issue fixed with fe83860bea7d59bf3119f623fd8d233f38b81472 binaries for your router are available in the feeds: run
```
ipkg update
ipkg upgrade
```

This is totally different issue from the originally reported, btw

**Comment by me11ved** [comment]
Yes indeed. This is another problem. Next time, create a new task.
Apache started now - but I could not start it in **fcgi** mode
Thank you.
